google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -1187,7 +1188,14 @@ public Builder setDeleteRules(Iterable<? extends DeleteRule> rules) {
 
     @Override
     public Builder setLifecycleRules(Iterable<? extends LifecycleRule> rules) {
-      this.lifecycleRules = rules != null ? ImmutableList.copyOf(rules) : null;
+      this.lifecycleRules =
+          rules != null ? ImmutableList.copyOf(rules) : ImmutableList.<LifecycleRule>of();
+      return this;
+    }
+
+    public Builder deleteLifecycleRules() {
+      setDeleteRules(null);
+      setLifecycleRules(null);
       return this;
```

Comment:
`@Override` and you need to introduce `public abstract Builder deleteLifecycleRules();` in the `BucketInfo.Builder` class.